### DISCUSSION TITLE
What is the recommended move-lang analyzer for VScode? (which is actively maintained)

### DISCUSSION THREAD

**Discussion by gftea** [aptos-labs/aptos-developer-discussions#231] [Dev Doc and Tutorial Feedback]
I follow the recommendation from aptos-dev https://aptos.dev/move/move-on-aptos, and install movebit  lang server. 
I tried to visit the movebit repo and search the extension, and it is not there
https://github.com/movebit?q=analyzer&type=all&language=&sort=

I wonder is it the officially recommended plugin by aptos?

**Comment by Alexzander-Stone** [comment]
Hey there, @gftea!

For VSCode, we recommend using the [aptos-move-analyzer](https://marketplace.visualstudio.com/items?itemName=MoveBit.aptos-move-analyzer) extension.

There's included [installation steps](https://marketplace.visualstudio.com/items?itemName=MoveBit.aptos-move-analyzer#:~:text=...-,Installation,-Note%3A) on the extension page, as well. 🫡

**Reply by gftea** [reply]
Hi, yes, I tried to use it, but I cannot find the open source repo for this extension, how can I give a feedback or to help to improve it?

**Reply by Alexzander-Stone** [reply]
Ah, I understand. There's a thread actively going on thanks to this, so we'll have updates on the docs/repo soon.

**Reply by gftea** [reply]
> The open source repo is here: https://github.com/movebit/move/tree/release/aptos-move-analyzer, and we will separate it to a single repo [movebit/aptos-move-analyzer] in the future

Thanks!
